.github/workflows/cd.yml

```
@@ -0,0 +1,50 @@
+name: backend deploy
+
+on:
+  push:
+    branches: [ "main" ]
```

테스트 할 수 있는 꿀팁!!

```suggestion
on:
  push:
    branches:
      - main
      - chore/ci-cd
```

위처럼 해두면 이거 테스트한다고 계속 main에 머지해보고 할 필요 없이 해당 브렌치에 테스트로 계속 푸시해보면서 해당 워크플로우 동작시켜서 배포 테스트 할 수 있어용

테스트 다 되고 마지막에 이거 머지시키기 전에 해당부분 `-chore/ci-cd` 부분 제거하고 머지시키면 되고용